sdk/lib/cmlib/cmlib.h

```diff
@@ -379,8 +379,12 @@ extern ULONG CmlibTraceLevel;
 //
 // Hack since big keys are not yet supported
 //
+#ifdef _BLDR_ 
+#define ASSERT_VALUE_BIG(h, s)
```

Comment:
Dunno whether you could just emit a DPRINT1 ? Unless it would spam the log and make booting crawl

Reply:
It only runs into vista's and XP's big keys during parsing